src/api/route-services/samples.js

```diff
@@ -9,19 +9,18 @@ class SamplesService {
     this.tableName = `samples-${config.clusterEnv}`;
   }
 
-  async getSamples(experimentId) {
-    const key = convertToDynamoDbRecord({
-      experimentId,
-    });
-
+  async getSamples(projectUuid) {
     const params = {
       TableName: this.tableName,
-      Key: key,
+      KeyConditionExpression: 'projectUuid = :projectUuid',
+      ExpressionAttributeValues: {
+        ':projectUuid': { S: projectUuid },
+      },
```

Comment:
It might make more sense to reuse the dynamodb marshall and unmarshall functions for this instead of manually formatting the input yourself

Reply:
Yea, you're right, it's a risk to do it manually. I'll change it to use AWS's marshall.